packages/ui/postcss.config.cjs

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
apparently this can go into the vite? config https://github.com/MutinyWallet/mutiny-web/pull/449

Reply:
there is no vite config since this uses rollup to be built. maybe vite would be a better way to build the package? the postcss is used only for storybook